x-pack/metricbeat/module/aws/elb/manifest.yml

```diff
@@ -16,3 +16,26 @@ input:
         statistic: ["Average"]
         name: ["Latency", "EstimatedALBActiveConnectionCount", "EstimatedALBConsumedLCUs", "EstimatedALBNewConnectionCount", "EstimatedProcessedBytes"]
         tags.resource_type_filter: elasticloadbalancing
+      - namespace: AWS/ApplicationELB
+        statistic: ["Sum"]
+        name: ["ActiveConnectionCount", "ClientTLSNegotiationErrorCount", "HTTP_Fixed_Response_Count",
+               "HTTP_Redirect_Count", "HTTP_Redirect_Url_Limit_Exceeded_Count",
+               "HTTPCode_ELB_3XX_Count", "HTTPCode_ELB_4XX_Count", "HTTPCode_ELB_5XX_Count",
+               "HTTPCode_ELB_500_Count", "HTTPCode_ELB_502_Count", "HTTPCode_ELB_503_Count",
+               "HTTPCode_ELB_504_Count", "IPv6ProcessedBytes", "IPv6RequestCount",
+               "NewConnectionCount", "ProcessedBytes", "RejectedConnectionCount",
+               "RequestCount", "RuleEvaluations"]
+      - namespace: AWS/ApplicationELB
+        statistic: ["Average"]
+        name: ["ConsumedLCUs"]
+      - namespace: AWS/NetworkELB
+        statistic: ["Average"]
+        name: ["ActiveFlowCount", "ActiveFlowCount_TLS", "ConsumedLCUs"]
+      - namespace: AWS/NetworkELB
```

Comment:
@jsoriano What do you think about the config here? This is the only way I can think of putting AWS/ELB, AWS/ApplicationELB and AWS/NetworkELB all in one metricset. If we use this, then users won't be able to specify what kind of ELB they'd like to collect metrics from.

Reply:
It looks good to me. The only thing is that we are going to have metrics that are the same thing, with different names (e.g. `HTTPCode_ELB_4XX_Count` and `HTTPCode_ELB_4XX`), but I guess we would have this problem also with different metricsets. We would need to add some support for processors in light modules to be able to rename fields in these cases.

>  If we use this, then users won't be able to specify what kind of ELB they'd like to collect metrics from.

I see this as a feature :slightly_smiling_face: Users only need to use this metricset for whatever load balancer they are using.

Reply:
Sounds good to me. Regarding to different names, it would be nice to have them being consistent. _But_ also the advantage of using cloudwatch metricset/light module here is to keep the metric names from Metricbeat exact same as whatever they are in AWS CloudWatch. Since we are not doing any processing to the metric names, if one day AWS decides to change the names, it won't break us either 😬 (not sure if it happens at all... )